File: experiment_jp_test/j2pyrawpy/for__1__COUNT_NUMBER_INCREASING_SUBSEQUENCES_SIZE_K.py
#!/usr/bin/env python
""" generated source for module for__1__COUNT_NUMBER_INCREASING_SUBSEQUENCES_SIZE_K """
class X(object):
    """ generated source for class X """
    @classmethod
    def numOfIncSubseqOfSizeK(cls, arr, n, k):
        """ generated source for method numOfIncSubseqOfSizeK """
        dp = [None]*k
        sum = 0
        i = k - 1
        while i < n:
            sum += dp[k - 1][i]
            i += 1
        return sum

File: experiment_jp_test/mutants/for__1__COUNT_NUMBER_INCREASING_SUBSEQUENCES_SIZE_K.java
static int numOfIncSubseqOfSizeK ( int arr [ ] , int n , int k ) { int dp [ ] [ ] = new int [ k ] [ n ] , sum = 0 ; for ( int i = k - 1 ; i < n ; i ++ ) { sum += dp [ k - 1 ] [ i ] ; } return sum ; }
